README.md

```diff
@@ -1,35 +1,22 @@
 # l3roamd
 
-l3roamd, pronunced *ɛl θriː ɹoʊm diː*, is a core part of layer 3 mesh networks.
-At first it will be built to work with [babeld](https://github.com/jech/babeld).
+[![standard-readme compliant](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
+
+> l3roamd, pronunced *ɛl θriː ɹoʊm diː*, is a core part of layer 3 mesh networks.
+
+l3roamd will will be built to work with [babeld](https://github.com/jech/babeld).
 
 Integrating tightly with mac80211 and netlink, l3roamd will be doing the following things:
  - monitor the presence of clients
  - allow network-wide search for IP-addresses
  - manage distribution of prefixes across the mesh (for router advertisements) [RA]
  - monitor babeld for duplicate host routes that this node also announces
- 
-Ideally, I'd like to split this up into multiple daemons communicating using standardized protocols:
-
-## Building l3roamd
 
-### Build
-in the root of this project, run:
- mkdir build
- cd build
- cmake ..
- make 
- make install
-
-to build and install the program.
+Ideally, this should be split up into multiple daemons communicating over standardized protocols.
```

Comment:
not any more. l3roamd does just client presence and client roaming. The other mentioned functionality (manage distribution of prefixes) has been implemented in prefixd.